src/components/BookCard.js

```diff
@@ -13,28 +13,29 @@ const BookCard = ({ books }) => {
       books.map((book) => (
         <div key={book.item_id} className="bookCard">
           <div className="book">
-            <p>{book.category}</p>
-            <h1>{book.title}</h1>
-            <p>{book.author}</p>
+            <p className="bookCategory">{book.category}</p>
+            <h1 className="bookTitle">{book.title}</h1>
+            <p className="bookAuthor">{book.author}</p>
             <div className="Book-btns">
-              <button type="button">Comments</button>
-              <div className="divide" />
-              <button onClick={() => { removeBook(book.item_id); }} type="button">Remove</button>
-              <div className="divide" />
-              <button type="button">Edit</button>
+              <button className="comment" type="button">Comments</button>
+              <div className="Line-2" />
+              <button className="remove" onClick={() => { removeBook(book.item_id); }} type="button">Remove</button>
+              <div className="Line-2" />
```

Comment:
- [ ] Please stick to the Zaplin design if you check dividers between comments and remove buttons the width is not matched with Zaplin Design so kindly fix this issue as well  :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-08-24 00-48-07](https://github.com/As1imwe-Mark/book_store/assets/45226848/5edaa85d-c48e-413e-bfae-53981a622416)

**Zaplin Design**
![Screenshot from 2023-08-24 00-47-41](https://github.com/As1imwe-Mark/book_store/assets/45226848/e9e9cf86-129c-45df-a01c-a3f290bc7855)